lib/themes/dosomething/paraneue_dosomething/js/components/ReactionsList.js

```diff
@@ -101,7 +101,8 @@ class ReactionsList extends React.Component {
 
     // Delete specified reaction record.
     if (reactionId) {
-      sendEvent('Onboarding', 'Un-like');
+      sendEvent('Onboarding', 'Un-like'); // Soon to be removed.
+      sendEvent('onboarding-v2', 'unlike', this.constructor.name);
```

Comment:
...by building for production! Either `ds assets` from the Vagrant machine, or `npm run build` depending on where you're building from.